src/util/positioning.ts

```diff
@@ -174,7 +174,7 @@ const positionService = new Positioning();
 export function positionElements(
     hostElement: HTMLElement, targetElement: HTMLElement, placement: string | Placement | PlacementArray,
     appendToBody?: boolean, baseClass?: string): Placement {
-  let placementVals: Array<Placement> = Array.isArray(placement) ? placement : [placement as Placement];
+  let placementVals: Array<Placement> = Array.isArray(placement) ? placement : placement.split(' ') as Array<Placement>;
```

Comment:
If we are opening the API that way, we might as well be more flexible regarding the white spacing. Using a regexp for instance: 

```suggestion
  let placementVals: Array<Placement> = Array.isArray(placement) ? placement : placement.split(/\s+/) as Array<Placement>;
```

Reply:
Thanks, @ymeine, good finding ! I'll just set the RegExp in a constant to avoid to get a new one each time.